palletjack-tools.gemspec

```diff
@@ -15,8 +15,10 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'kvdag', '~> 0.0.3'
   s.add_runtime_dependency 'dns-zone', '~> 0.3'
   s.add_runtime_dependency 'ruby-ip', '~> 0.9'
+  s.add_runtime_dependency 'rugged', '~> 0.24'
   s.files	= [ 'README.md', 'LICENSE' ]
   s.files	+= Dir['tools/*']
+  s.files	+= [ 'lib/palletjack/tool-support.rb' ]
```

Comment:
Actually, I think it already is included there, since `palletjack.gemspec` says:
```ruby
  s.files	+= Dir['lib/**/*.rb']
```
But I'll remove it from `palletjack-tools`.